scripts/build-package

```diff
@@ -2,6 +2,8 @@
 set -e
 
 pip install --require-hashes -r requirements.txt --target ./package
+rm -rf ./package/cdn_definitions*
```

Comment:
The implication here seems to be that if you want to use the private data, you're supposed to uninstall cdn-definitions.  So you can have cdn-definitions or cdn-definitions-private installed, but not both.  Is that right?

If so - I don't think it should be that way.  I think it should be more like: if you're going to use any cdn-definitions dataset, then you install cdn-definitions and that gives you reference data, plus functionality common to any dataset, such as the expected schema for the dataset and maybe a handful of utility functions.  Then you can install cdn-definitions-private to *supplement* that with the desired private dataset - not *replace* it.  If you have to uninstall cdn-definitions to install cdn-definitions-private then that implies you lose the common functionality (such as the schema) or that cdn-definitions-private has to duplicate it.

Should we maybe work on a brief design doc explaining a bit more the scope of each project?  Seems like it might not be entirely clear.